### DISCUSSION TITLE
TV Asahi

### DISCUSSION THREAD

**Discussion by inklingboi08** [iptv-org/community#946] [Q&A]
### Channel ID

JOEXDTV.jp

### Stream URL (optional)

https://aqstream.com/jp/tva/TV-Asahi

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19850
